src/services/itemMembership/service.ts

```diff
@@ -46,7 +46,11 @@ export class ItemMembershipService {
         ${this.mailerService.buildButton(link, t(MAIL.SHARE_ITEM_BUTTON))}
       `;
 
-    const title = t(MAIL.SHARE_ITEM_TITLE, { creatorName: actor.name, itemName: item.name });
+    const title = t(MAIL.SHARE_ITEM_TITLE, {
+      creatorName: actor.name,
+      itemName: item.name,
+      interpolation: { escapeValue: false },
```

Comment:
I'm just wondering if we have to escape some html or not ? I know that recent mail clients protect us from malicious html, but maybe it could be an added protection to at least replace common html like the `<>` ? What do you think ?

Reply:
Yes, I thought about it too. In this particular case it should be fine since the mail subject should always be treated as plain text by the email clients.

But it's definitely a good practice to sanitize user input. But if we want to sanitize the input in this case, we would have to roll out our own sanitization function which would remove the `<>"` but not the `'`. Seems like a bit of an overkill just for the email subject to me.

What do your thoughts ?